api-settings/releases/4.4.3-zulu/index.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Release": "4.4.3-zulu",
+  "Date": "10-Jun-2023 00:00",
+  "Label": "pre-release",
+  "Type": "Unstable",
+  "ReleaseUrl": "",
+  "CloudBuild": true,
+  "UnifiedConfig": true,
+  "Tag": "4.4.3",
```

Comment:
```suggestion
  "Tag": "4.4-maintenance",
```

Reply:
Adding as tag relates to the tag on the docker image to be pulled from the cloudbuild repository at docker Hub.

Reply:
Updated.